lms/templates/discovery/course_card.underscore

```diff
@@ -3,7 +3,7 @@
         <header class="course-image">
             <div class="cover-image">
                 <img src="<%- image_url %>" alt="<%- content.display_name %> <%- content.number %>" />
-                <div class="learn-more" aria-hidden=true><%- gettext("LEARN MORE") %></div>
+                <div class="learn-more" aria-hidden="true"><%- gettext("LEARN MORE") %></div>
```

Comment:
@cahrens This is a weird case and frankly the author could have made better choices. But, these are the course cards on the listing page. When you hover over one, the image is covered up with the "Learn more" button. But since the entire card is wrapped in a link, the "learn more" is unnecessary.

This would read as a link to the course name with other relevant information such as the course number, start times, etc. The fact that it reads as a link on the course listing page is clear. Again, kinda funky choices, but it works.